pkgs/tools/security/otpgen/default.nix

```diff
@@ -0,0 +1,40 @@
+{ 
+  pkgs
+  , lib
+  , stdenv
+  , makeWrapper
+  , fetchFromGitHub
+  , cmake
+  , pkg-config
+  , cryptopp
+  , qtbase
+}:
+
+stdenv.mkDerivation rec {
+
+  pname = "OTPGen";
+  version = "0.9.3";
+
+  src = fetchFromGitHub {
+    owner = "magiruuvelvet";
+    repo = "OTPGen";
+    rev = "v${version}";
+    sha256 = "19nyr3gmfxak0zdzibs1pjcdaxyy8x7jvrhgi3bsg9jbgf1m6xsr";
+  };
+
+  patches = [ ./core-cmake-cxx.patch ./cmakelists.patch ];
```

Comment:
Could you add comment explaining what it does and where it came from to each patch?

Reply:
Fixed here : https://github.com/NixOS/nixpkgs/pull/87887/commits/9228a2d6e7c562348bebfbb24983a20a0f9b1809, removed unnecessary patches, commented in default.nix.